SPECS-EXTENDED/gspell/gspell.spec

```diff
@@ -78,12 +79,19 @@ find $RPM_BUILD_ROOT -name '*.la' -delete
 %{_libdir}/pkgconfig/gspell-1.pc
 %{_datadir}/gir-1.0/
 %{_datadir}/vala/
+%{_datadir}/installed-tests/%{name}-1/
```

Comment:
We don't need test files as they'll unnecessarily bloat the rpm size.

Reply:
I will exclude those files.